baseline/OSPS-DO.yaml

```diff
@@ -100,15 +100,30 @@ criteria:
       The project documentation MUST include a
       descriptive statement about the scope and
       duration of support.
-    rationale: # TODO
-    # TODO: Integrate with advice from https://endoflife.date/recommendations
+    rationale: |
+      Stating the support expecations the project
+      has (in this case when the project will no
+      longer be actively supporting the project and 
+      considering it "end of life") are important
```

Comment:
Do we mean the project or versions of a project?

I expect that most projects, like most people, don't expect the end until it happens.

Reply:
hrm....  yes.... both versions and/or project closes up.

Reply:
Thinking about it... I wouldn't expect (for example) https://github.com/angular/angular to have a "we'll stop supporting the project on $DATE" until the project was already effectively dead.

I _would_ expect them to have a "we'll stop supporting 6.x on $DATE", but that's a more nuanced statement.

Reply:
I don't want to be prescriptive on what support of comms channels projects must use. With implementation we can enhance the example to help projects understand better what the objective is and how their particular flavour may meet that requirement.  As we get these criteria shored up, we should identify those things that lend themselves to being made into a template, so that folks could easily copy and edit them to meet their needs.